src/components/layout/Templates/BoardLayout.tsx

```diff
@@ -243,6 +243,12 @@ const BackgroundImage = () => {
     return null;
   }
 
+  // Check if the background image URL is a video
+  const videoFormat = /\.(mp4|webm|ogg)$/i.exec(config.settings.customization.backgroundImageUrl)?.[1];
+  if (videoFormat) {
+    return BackgroundVideo(config.settings.customization.backgroundImageUrl, videoFormat);
```

Comment:
Use a react component instead of the `BackgroundVideo` function by changing the arguments of it to an object containing video and videoFormat:

```tsx
interface BackgroundVideoProps {
  videoSource: string;
  videoFormat: string;
}

const BackgroundVideo = ({video, videoFormat}: BackgroundVideoProps ) => {
  // Your implementation
}
```

Usage: `<BackgroundVideo videoSource={config.settings.customization.backgroundImageUrl} videoFormat={videoFormat} />`